projects/igniteui-angular/migrations/common/tsLogger.ts

```diff
@@ -56,52 +22,31 @@ export class Logger implements ts.server.Logger {
         this.msg(s, ts.server.Msg.Info);
     }
 
-    public err(s: string) {
-        this.msg(s, ts.server.Msg.Err);
-    }
-
-    public startGroup() {
-        this.inGroup = true;
-        this.firstInGroup = true;
-    }
-
-    public endGroup() {
-        this.inGroup = false;
-    }
-
-    public loggingEnabled() {
-        return !!this.logFilename || this.traceToConsole;
-    }
-
-    public hasLevel(level: ts.server.LogLevel) {
-        return this.loggingEnabled() && this.level >= level;
-    }
-
     public msg(s: string, type: ts.server.Msg = ts.server.Msg.Err) {
-        if (!this.canWrite) { return; }
-
-        s = `[${nowString()}] ${s}\n`;
-        if (!this.inGroup || this.firstInGroup) {
-            const prefix = Logger.padStringRight(type + ' ' + this.seq.toString(), '          ');
-            s = prefix + s;
+        if (type === ts.server.Msg.Info) {
+            // TODO: Info ignored for now, hook into context in the future
+            // console.log(s);
```

Comment:
@jackofdiamond5 I've ignored those for now, because they are a bit chatty with useless info for the scope:
```log
> ng g ./node_modules/igniteui-angular/migrations/migration-collection.json:migration-17
    Applying migration for Ignite UI for Angular to version 10.2.0
Format host information updated
reload projects.
Before ensureProjectForOpenFiles:
After ensureProjectForOpenFiles:
Host file extension mappings updated
    Installing temporary migration dependencies via yarn.
...
```